### PR TITLE
Test fix for cert names

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

File: okhttp/src/test/java/okhttp3/internal/tls/HostnameVerifierTest.java [hunk 1]
```diff
@@ -373,7 +380,11 @@
         + "-----END CERTIFICATE-----\n");
 
     X509Certificate peerCertificate = ((X509Certificate) session.getPeerCertificates()[0]);
-    assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com", "*.������.co.jp");
+    if (platform.isConscrypt()) {
+      assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com");
+    } else {
+      assertThat(certificateSANs(peerCertificate)).containsExactly("*.bar.com", "*.������.co.jp");
```
Comment (swankjesse) [inline okhttp/src/test/java/okhttp3/internal/tls/HostnameVerifierTest.java:386]: is this asserting a bug? Shouldn’t the cert library be limiting these to ASCII strings?

Comment (swankjesse) [reply okhttp/src/test/java/okhttp3/internal/tls/HostnameVerifierTest.java:386]: (the bug is in JSSE, Coretto, etc.)

Comment (yschimke) [reply okhttp/src/test/java/okhttp3/internal/tls/HostnameVerifierTest.java:386]: Yep.

**Comment by yschimke** [comment]
@swankjesse sorry, misread as approved, instead of just a comment.  Let me know if you want me to revert.

**Pull request merged**